selfservice/flow/verification/handler.go

```diff
@@ -361,7 +375,8 @@
 //	  200: selfServiceVerificationFlow
 //	  303: emptyResponse
 //	  400: selfServiceVerificationFlow
-//	  410: jsonError
+//	  410: selfServiceFlowReplacedError
+//	  410: selfServiceFlowExpiredError
```

Comment:
Well, kind of :D both errors can occur here, because `selfServiceFlowReplacedError` does not support the same fields that `selfServiceFlowExpiredError` does. So I can't just replace it like that. Keeping both is not possible?

Reply:
I think it will choose one of the two anyways. Not sure how to solve this to be honest. I think we need an error model that supports this in general. Not sure how well polymorphism works for responses though ...

Reply:
Or, actually, I'm not sure. Maybe this works, you have to check the schema output.